.github/workflows/CI.yml

```diff
@@ -473,6 +479,17 @@ jobs:
           cat build/packages/VERSION
           echo "::set-output name=version::$Version"
 
+      - name: Setting branch for testing
+        id: get_sha
+        run: |
+          # Checking whether it is for integration testing or for the merge pull request from CI
+          commit_id="$CI_DEFAULT_BRANCH"
+          if [ "${{ github.event.inputs.sha }}" != "" ]
+          then
+            commit_id = "${{ github.event.inputs.sha }}"
+          fi
+          echo "::set-output name=commit_id::$commit_id"
```

Comment:
Will this work on release branches?

Reply:
Yes. Changed the default commit id to use **GITHUB_REF_NAME** would work not only on release branch, but also the main and the dev branch as well.